src/pages/MyEarnings/ElasticPools/SinglePool/index.tsx

```diff
@@ -0,0 +1,542 @@
+                }}
+                to={
+                  currency0Slug && currency1Slug
+                    ? `/${NETWORKS_INFO[chainId].route}${APP_PATHS.ELASTIC_CREATE_POOL}/${currency0Slug}/${currency1Slug}/${feeAmount}`
+                    : '#'
+                }
+              >
+                + <Trans>Add Liquidity</Trans>
+              </ButtonLight>
+            )}
+
+            <Flex sx={{ gap: '0.75rem' }}>
+              <ButtonIcon
+                style={{
+                  width: '36px',
+                  height: '36px',
+                }}
+                as="a"
+                href={analyticUrl}
+                target="_blank"
+                onClick={e => {
+                  e.stopPropagation()
+                }}
+              >
+                <BarChart2 color={theme.subText} />
+              </ButtonIcon>
+
+              <ButtonIcon
+                style={{
+                  width: '36px',
+                  height: '36px',
+                  transform: isExpanded ? 'rotate(-180deg)' : undefined,
+                  transition: 'transform 150ms ease',
+                }}
+                disabled={!pool}
+                onClick={toggleExpanded}
+              >
+                {!pool ? <Loader /> : <DropdownSVG />}
+              </ButtonIcon>
+            </Flex>
+          </Flex>
+        </MobileStatWrapper>
+        {isExpanded && mobileView && <Divider />}
+        {isExpanded && isExpandable && positions}
+      </Box>
+    )
+  }
+
+  return (
+    <Wrapper>
+      <Row onClick={() => isExpandable && setExpanded(e => !e)}>
+        <Flex alignItems="center">
+          <DownIcon isOpen={isExpanded} role="button" />
+          <Box sx={{ position: 'relative' }}>
+            <DoubleCurrencyLogo currency0={visibleCurrency0} currency1={visibleCurrency1} size={32} />
+            <img
+              src={NETWORKS_INFO[chainId].icon}
+              alt={NETWORKS_INFO[chainId].name}
+              width={18}
+              height={18}
+              style={{
+                position: 'absolute',
+                bottom: 0,
+                right: 0,
+                zIndex: 1,
+              }}
+            />
+          </Box>
+          <Flex flexDirection="column" sx={{ gap: '4px' }}>
+            <Flex
+              sx={{
+                alignItems: 'center',
+                gap: '8px',
+              }}
+            >
+              <Text>{poolName}</Text>
+              <Badge $color={theme.blue}>FEE {feePercent}</Badge>
+              {isFarmingPool && <FarmTag noText address={poolEarning.id} chainId={chainId} />}
+            </Flex>
+
+            {share}
+          </Flex>
+        </Flex>
+        <Text>{formatDollarAmount(+poolEarning.totalValueLockedUsd)}</Text>
+
+        <MouseoverTooltip
+          width="fit-content"
+          placement="top"
+          text={
+            <APRTooltipContent
+              farmAPR={farmAPR}
+              farmV2APR={
+                // TODO
```

Comment:
todo

Reply:
yes. farm v2 is not integrated yet. waiting for kn protocol